main/src/library/Foldable.flix

```diff
@@ -97,7 +105,7 @@ pub class Foldable[t : Type -> Type] {
     /// Returns `None` if `t` is empty.
     ///
     pub def findRight(f: a -> Bool, t: t[a]): Option[a] =
-        Foldable.foldRight((x, k) -> Option.withDefault(default = if (f(x)) Some(x) else None, k()), None, t)
+        Foldable.foldRight((x, acc) -> Option.withDefault(default = if (f(x)) Some(x) else None, acc), None, t)
```

Comment:
Actually these should be expressed using foldRightWithCont. But maybe you change that in a subsequent PR?